workspace/src/common/launch/art_localization_launch/launch/art_localization.launch.py

```diff
@@ -50,7 +50,7 @@ def generate_launch_description():
     AddLaunchArgument(ld, "art_localization/input/accelerometer", "/sensing/accelerometer/data")
     AddLaunchArgument(ld, "art_localization/input/vehicle_inputs", "/control/vehicle_inputs")
     AddLaunchArgument(ld, "art_localization/input/ground_truth", "/vehicle/state")
-    AddLaunchArgument(ld, "art_localization/output/vehicle_state", "/vehicle/filtered_state")
+    AddLaunchArgument(ld, "art_localization/output/vehicle/filtered_state", "/vehicle/filtered_state")
```

Comment:
necessary? and what does filtered state mean? is this consistent with the state estimation node?